src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -96,68 +119,93 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                         notifications.ShowError(error?.Message ?? ex.Message);
                         return Observable.Empty<IPullRequestModel>();
                     }));
+            isExecuting = CreatePullRequest.IsExecuting.ToProperty(this, x => x.IsExecuting);
```

Comment:
:+1: i guess this will become clear when the `IsBusy` state is actually updated.